common/internal_model/src/test/scala/uk/ac/wellcome/models/work/generators/InstantRangeGenerators.scala

```diff
@@ -0,0 +1,19 @@
+package uk.ac.wellcome.models.work.generators
+
+import java.time.{LocalDateTime, ZoneOffset}
+
+import uk.ac.wellcome.models.work.internal._
+
+trait InstantRangeGenerators {
+  // This is used to make the tests more readable.
```

Comment:
While true, this comment isn't adding much – we have dozens of `generators/create` methods lying around, none of which have this comment.